src/main/java/gameframework/game/GameConfiguration.java

```diff
@@ -13,20 +13,44 @@
 
 public class GameConfiguration {
 
+	protected int nbRows;
+	protected int nbColumns;
+	protected int spriteSize;
+	protected int nbLives;
+
+	// CONSTRUCTORS
+
+	public GameConfiguration(int nbRows, int nbColumns, int spriteSize,
+			int nbLives) {
+		this.nbRows = nbRows;
```

Comment:
it would be nice if the user of this constructor could only set a subset of these variables. As it is, the user has to provide either all of the values or none. To do so, you could add more constructors and/or consider that if an argument has value 0, the default value would be used instead.